http-2.gemspec

```diff
@@ -11,5 +11,7 @@ Gem::Specification.new do |spec|
   spec.license       = 'MIT'
   spec.files         = Dir["LICENSE", "README.md", "lib/**/*.rb"]
 
+  spec.add_runtime_dependency 'base64'
```

Comment:
Should we do this in a different minor version from the other fixes? (in particular I'm thinking, if this causes issues for some users, being able to use the smaller version of the gem would be useful).

Reply:
I think it's fine to not do that. The current experience is already what's expected, and a new version can be a carrot to upgrade (using new ruby, smaller gem, etc..)